noir-projects/noir-protocol-circuits/crates/private-kernel-lib/src/components/private_kernel_circuit_output_validator.nr

```diff
@@ -390,20 +368,17 @@ impl PrivateKernelCircuitOutputValidator {
             offsets.unencrypted_logs_hashes,
             storage_contract_address
         );
-        validate_array_appended_reversed_scoped(
+        validate_array_appended(
```

Comment:
Unfortunately this has increased 4k. I think there's some unreasonably heavy stuffs going on so we should be able to reduce it a bit. (Will refactor in another pr!)